recipes/slycot/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slycot" %}
+{% set version = "0.2.0" %}
+{% set md5 = "35c1737180ecee67ba6574d65dde85b1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 200
```

Comment:
I've always had to set this to 200 for things that required openblas. Is this now changed?